packages/utils/src/webpack/selectors.js

```diff
@@ -79,6 +79,30 @@ const getAssetsMetrics = (job) => get(job, 'metrics.webpack.assets', {});
  */
 const getModulesMetrics = (job) => get(job, 'metrics.webpack.modules', {});
 
+/**
+ *
+ * Select webpack module size including duplication
+ *
+ * @param {Object} job Job data
+ * @param {Object} job.metrics Job metrics
+ * @param {Object} job.metrics.webpack Job webpack metrics
+ * @param {Object} job.metrics.webpack.modules Job webpack module metrics
+ *
+ * @return {Object} Webpack module metrics
+ */
+export const getModuleDuplicateSize = (job) => {
```

Comment:
This works as expected with the existing fixtures, but in the past we had performance issues with reduce and object spread on large sets.

If you want to have a try with larger payloads, here is a project with almost 4k modules:
- current: https://app.relative-ci.com/projects/grRdPUlg9DQleoTLUjey/jobs/10476-sTEZhlNQuEOgv8visQSU?info=1
- baseline: https://app.relative-ci.com/projects/grRdPUlg9DQleoTLUjey/jobs/7565-CNp0JK5PY6COdnF1JC6J?info=1

Reply:
I decided to measure the performance (in Chrome) with those 4k module files and it seems to be a non-issue (this whole function takes 1-3ms). In practice drawing the UI seems to take about 2 orders of magnitude longer than mutating the numbers.

![image](https://github.com/relative-ci/bundle-stats/assets/26855/1d04ce34-e40c-4d5e-bfaa-b185324af365)

Reply:
Here was the diff for timing:

![image](https://github.com/relative-ci/bundle-stats/assets/26855/da06d560-bf59-4743-adf1-d076287bef55)

Reply:
Thanks for checking @danielbeardsley!